mesheryctl/internal/cli/root/perf/apply.go

```diff
@@ -35,28 +36,35 @@ var (
 )
 
 var applyCmd = &cobra.Command{
-	Use:   "apply",
+	Use:   "apply [profile-name | --profile | --file] --flags",
 	Short: "Run a Performance test",
 	Long:  `Run Performance test using existing profiles or using flags`,
 	Args:  cobra.MinimumNArgs(0),
 	Example: `
-	Execute a Performance test with the specified performance profile
-	mesheryctl perf apply <profile name> --flags
+// Execute a Performance test with the specified performance profile
+mesheryctl perf apply meshery-profile --flags
 
-	Execute a Performance test without a specified performance profile
-	mesheryctl perf apply --profile <profile-name> --url <url>
+// Execute a Performance test with creating a new performance profile
+mesheryctl perf apply --profile meshery-profile-new --url <url>
 
-	Run Performance test using SMP compatible test configuration
-	mesheryctl perf apply -f <filepath>
+// Run Performance test using SMP compatible test configuration
+mesheryctl perf apply -f <filepath>
```

Comment:
`-f` is not a shorthand for `--file` here. We are using `-f` shorthand on other commands like `pattern` so we can also add it here.

Reply:
![Screenshot 2021-09-24 at 3 21 30 AM](https://user-images.githubusercontent.com/49474499/134589023-a97ed5c8-bece-4355-8284-a6245bac7601.png)

Reply:
this one is in the other PR.